server/router/driver.ts

```diff
@@ -29,23 +33,33 @@ router.get('/:id/profile', (req, res) => {
 });
 
 // Put a driver in Drivers table
-router.post('/', (req, res) => {
+router.post('/', validateUser('Driver'), (req, res) => {
   const { body } = req;
   const driver = new Driver({
     ...body,
-    id: uuid()
+    id: uuid(),
+  });
+  db.create(res, driver, (data: DriverType) => {
+    const { locals: { user: { id } } } = res;
+    if (id) {
+      res.send(data);
+    } else {
+      res.send({
+        ...data,
+        jwt: jwt.sign({ id: data.id, userType: 'Driver' }, process.env.JWT_SECRET!),
+      });
+    }
```

Comment:
This could apply to all of the create endpoints. You mentioned that the jwt id would be null upon creation but that the user could get it later.

This should not matter because a dispatcher will always be the one to create new users of any type. Since this is the case, you know that the person creating the users' accounts are not the users themselves and therefore should never get those users' jwt tokens returned to them.